src/sentry/rules/actions/integrations/base.py

```diff
@@ -31,46 +31,44 @@ def integration_key(self) -> str:
         pass
 
     def is_enabled(self) -> bool:
-        enabled: bool = self.get_integrations().exists()
+        enabled: bool = bool(self.get_integrations())
         return enabled
 
     def get_integration_name(self) -> str:
         """Get the integration's name for the label."""
-        try:
-            integration = self.get_integration()
-        except Integration.DoesNotExist:
+        integration = self.get_integration()
+        if not integration:
             return "[removed]"
 
         _name: str = integration.name
         return _name
 
-    def get_integrations(self) -> QuerySet[Integration]:
-        query: QuerySet[Integration] = Integration.objects.get_active_integrations(
```

Comment:
Should we remove this method now?

Reply:
Good call -- I can collapse this into callsites.

Reply:
Ah, kind of hard to remove this because subclasses actually override this and provide additional filtering, so it must remain as part of that protocol.